sessionctx/binloginfo/binloginfo_test.go

```diff
@@ -380,9 +380,9 @@ func mutationRowsToRows(c *C, mutationRows [][]byte, columnValueOffsets ...int)
 }
 
 func (s *testBinlogSuite) TestBinlogForSequence(c *C) {
-	c.Assert(failpoint.Enable("github.com/pingcap/tidb/store/tikv/mockSyncBinlogCommit", `return(true)`), IsNil)
+	c.Assert(tikv.MockSyncBinlogCommit.Enable(`return(true)`), IsNil)
 	defer func() {
-		c.Assert(failpoint.Disable("github.com/pingcap/tidb/store/tikv/mockSyncBinlogCommit"), IsNil)
+		tikv.MockSyncBinlogCommit.Disable()
```

Comment:
should we assert it is nil?

Reply:
I think `Disable()` will always succeed unless it's not enabled.